platform/tron/api.go

```
@@ -40,14 +40,45 @@ func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
 	return txs, nil
 }
 
-/// Normalize converts a Tron transaction into the generic model
-func Normalize(srcTx *Tx) (tx blockatlas.Tx, ok bool) {
-	if len(srcTx.Data.Contracts) < 1 {
-		return tx, false
+func (p *Platform) GetTokenTxsByAddress(address, token string) (blockatlas.TxPage, error) {
+	tokenTxs, err := p.client.GetTxsOfAddress(address, token)
+	if err != nil {
+		return nil, err
+	}
+
+	var tokenInfo *Asset
+	if token != "" {
+		var wg sync.WaitGroup
+		wg.Add(1)
+			go func(id string) {
+				defer wg.Done()
+				info, err := p.client.GetTokenInfo(id)
+				if err != nil {
+					logger.Error(err)
```

I think we need to return this error to the function with `GetTokenInfo` fail, right?